src/masternode.h

```diff
@@ -224,6 +235,8 @@ class CMasternode : public masternode_info_t
         READWRITE(addr);
         READWRITE(pubKeyCollateralAddress);
         READWRITE(pubKeyMasternode);
+        READWRITE(keyIDCollateralAddress);
```

Comment:
Should probably bump `CMasternodeMan::SERIALIZATION_VERSION_STRING` (masternodeman.cpp#L29) because of changes in serialization.